xmtp_mls/src/groups/mod.rs

```diff
@@ -468,13 +467,25 @@ impl MlsGroup {
         ApiClient: XmtpApi,
     {
         let account_addresses = sanitize_evm_addresses(account_addresses_to_add)?;
-        let inbox_id_map = client.api_client.get_inbox_ids(account_addresses).await?;
+        let inbox_id_map = client
+            .api_client
+            .get_inbox_ids(account_addresses.clone())
+            .await?;
         // get current number of users in group
         let member_count = self.members()?.len();
         if member_count + inbox_id_map.len() > MAX_GROUP_SIZE as usize {
             return Err(GroupError::UserLimitExceeded);
         }
 
+        if inbox_id_map.len() != account_addresses.len() {
```

Comment:
The way I am doing this validation is not very good. I have to jump through some hoops to get the actual list of missing addresses.